C/tests/c4QueryTest.cc

```diff
@@ -415,15 +415,8 @@ N_WAY_TEST_CASE_METHOD(C4QueryTest, "C4Query dict literal", "[Query][C]") {
 N_WAY_TEST_CASE_METHOD(C4QueryTest, "C4Query N1QL parse error", "[Query][C][N1QL][!throws]") {
```

Comment:
There is no more error here so perhaps this test should be renamed.

Reply:
I see. Let me just remove it altogether.